src/database/schema/ChatSchema.ts

```diff
@@ -519,4 +525,8 @@ class ChatSchema implements BaseDbSchema {
   };
 }
 
+function isSavingAttachmentAllowed(messageType): boolean {
+  return ![MESSAGE_TYPE_REPLY_PROMPT, MESSAGE_TYPE_DELETED].includes(messageType);
+}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function `isSavingAttachmentAllowed` has been added. This function checks if the message type allows saving attachments, which improves code readability and maintainability.
```diff
+ function isSavingAttachmentAllowed(messageType): boolean {
+   return ![MESSAGE_TYPE_REPLY_PROMPT, MESSAGE_TYPE_DELETED].includes(messageType);
+ }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->